### PR TITLE
10.0 mig purchase order type

### PR DESCRIPTION
**Pull request by kikosanchez**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together "OCA transbot" commits using this guide https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests and check Travis fail

File: purchase_order_type/README.rst [hunk 1]
```diff
@@ -1,38 +1,53 @@
...
 To use this module, you need to:
 
 * Attribute a type when editing purchase orders
+* Order Type will be autofilled when selecting a partner that has a type set
+  in its data.
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/142/8.0
+   :target: https://runbot.odoo-community.org/runbot/142/9.0
```
Comment (LoisRForgeFlow) [inline purchase_order_type/README.rst]: 10.0

File: purchase_order_type/__manifest__.py [hunk 2]
```diff
@@ -1,38 +1,30 @@
...
+# Copyright 2017 Kiko Sanchez <kiko@comunitea.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order Type",
+    "version": "10.0.1.0.0",
+    "category": "Purchase Management",
+    "website": "http://www.camptocamp.com",
+    "author": "Camptocamp,"
+              "Tecnativa,"
+              "Odoo Community Association (OCA)"
```
Comment (LoisRForgeFlow) [inline purchase_order_type/__manifest__.py]: Space and comma: `"Odoo Community Association (OCA), "`

Comment (LoisRForgeFlow) [reply purchase_order_type/__manifest__.py]: Also travis is complaning of this, can it be that OCA must be the last author?

File: purchase_order_type/__manifest__.py [hunk 3]
```diff
@@ -1,38 +1,30 @@
...
+              "Comunitea",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "purchase",
+    ],
+
```
Comment (LoisRForgeFlow) [inline purchase_order_type/__manifest__.py]: remove blank line

File: purchase_order_type/models/purchase_order.py [hunk 4]
```diff
@@ -1,27 +1,11 @@
...
-
-from openerp import models, fields, api
-from openerp.addons.purchase.purchase import purchase_order
+# Copyright 2015 Guewen Baconnier <guewen.baconnier@camptocamp.com>
+# Copyright 2016 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2017 Kiko Sanchez <kiko@comunitea.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from odoo import api, fields, models
+from odoo.addons.purchase.models.purchase import PurchaseOrder as purchase_order
```
Comment (LoisRForgeFlow) [inline purchase_order_type/models/purchase_order.py]: Extra line here to make travis a happy friend! :wink:

**Review by LoisRForgeFlow — COMMENTED** [review]
Code review. Some minor comments.

Thanks for the migration

**Review by MiquelRForgeFlow — APPROVED** [review]
code review :+1: 
Please squash your commits, and also OCA translation commits

**Comment by lasley** [comment]
@kikosanchez - I think this is ready for merge, but just waiting on the commit squashing that was mentioned in https://github.com/OCA/purchase-workflow/pull/403#issuecomment-304750294 and https://github.com/OCA/purchase-workflow/pull/403#pullrequestreview-45712077. Did you plan to attend to that?

**Pull request closed without merging**